src/components/DropdownItem.js

```diff
@@ -0,0 +1,374 @@
+            transition: fill 75ms linear;
+          }
+        }
+
+        .drui-dropdownItem__description {
+          flex: 0 0 100%;
+          font-size: 12px;
+          line-height: 15px;
+          text-align: left;
+          color: ${defaultTheme.dropdown.description.color};
+        }
+
+        .theme--dark {
+          .drui-dropdownItem.drui-dropdownItem--disabled {
+            .drui-dropdownItem__action {
+              &:hover {
+                cursor: not-allowed;
+                background-color: transparent;
+              }
+            }
+
+            .drui-dropdownItem__icon,
+            .drui-dropdownItem__title,
+            .drui-dropdownItem__titleIcon,
+            .drui-dropdownItem__description {
+              opacity: 0.6;
+              color: ${darkTheme.dropdown.item.disabledColor};
+              fill: ${darkTheme.dropdown.item.disabledColor};
+            }
+
+            &:hover {
+              .drui-dropdownItem__icon,
+              .drui-dropdownItem__title,
+              .drui-dropdownItem__titleIcon,
+              .drui-dropdownItem__description {
+                opacity: 0.6;
+                color: ${darkTheme.dropdown.item.disabledColor};
+                fill: ${darkTheme.dropdown.item.disabledColor};
+              }
+            }
+          }
+
+          .drui-dropdownItem__action {
+            &:hover {
+              background-color: ${darkTheme.dropdown.item.hoverColor};
+
+              .drui-dropdownItem__title {
+                color: ${darkTheme.dropdown.item.title.hoverColor};
+              }
+
+              .drui-dropdownItem__icon {
+                fill: ${darkTheme.dropdown.item.title.hoverColor};
+              }
+
+              .drui-dropdownItem__titleIcon {
+                fill: ${darkTheme.dropdown.item.title.hoverIconFill};
+              }
+            }
+
+            &:focus {
+              background-color: ${darkTheme.dropdown.item.hoverColor};
+            }
+
+            &:active {
+              background-color: ${darkTheme.dropdown.item.activeColor};
+            }
+          }
+
+          .drui-dropdownItem__icon {
+            fill: ${darkTheme.dropdown.item.iconColor};
+          }
+
+          .drui-dropdownItem__title {
+            color: ${darkTheme.dropdown.item.title.color};
+          }
+
+          .drui-dropdownItem__titleIcon {
+            fill: ${darkTheme.dropdown.item.title.iconFill};
+          }
+
+          .drui-dropdownItem__description {
+            color: ${darkTheme.dropdown.description.color};
+          }
+        }
+      `}</style>
+    </div>
+  );
+};
+
+DropdownItem.propTypes = {
+  title: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.shape({
+      text: PropTypes.string.isRequired,
+      Icon: PropTypes.node.isRequired,
+    }),
+  ]).isRequired,
+  description: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.node,
+  ]),
+  Icon: PropTypes.oneOfType([
+    PropTypes.node,
```

Comment:
I wouldn't say that `PropType.node` fits in this place.
We expect to have a React component here and we use this props only like this.